modules/engine/src/animation-loop/animation-loop.ts

```diff
@@ -304,7 +304,7 @@ export class AnimationLoop {
   }
 
   _cancelAnimationFrame() {
-    if (this._animationFrameId !== null) {
+    if (this._animationFrameId == null) {
```

Comment:
Nit: We use `===` for comparisons.

Reply:
@ibgreen  The reason for using '==' is that it can compare null and 'undefined'

Reply:
@ibgreen The reason for using '==' is that it can compare `null` and `undefined`

Reply:
Yes I understand, but for that case, we just use boolean negation. I find the difference between `===` and `==` too subtle and a cause of hard to spot errors, so we just avoid `==`. But that is of course just our preference in this particular code base.